docs/usage/executor/kubectl.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
this is moved from https://github.com/kubeshop/botkube-docs/blob/main/docs/usage/executing-commands.md